hikari/impl/rest.py

```diff
@@ -3214,7 +3214,8 @@ def command_builder(
     ) -> special_endpoints.ContextMenuCommandBuilder:
         ...
 
-    def command_builder(
+    @deprecation.deprecated("2.0.0.dev106", "slash_command_builder or context_menu_builder")
+    def command_builder(  # type: ignore[override]
```

Comment:
Similar thing here for reverting it to the signature (including removing the overloads) and behaviour found on #master while deprecating it to avoid any needless breaking changes here